README.md

```diff
@@ -4,75 +4,46 @@
 
 ![Screenshot of the Gutenberg Editor, editing a post in WordPress](https://cldup.com/R84R5fNgrI.png)
 
-This repo is the development hub for the <a href="https://make.wordpress.org/core/2017/01/04/focus-tech-and-design-leads/">editor focus in WordPress Core</a>. `Gutenberg` is the project name.
+Welcome to the development hub for the WordPress Gutenberg Project!
 
-## Getting started
-- **Download:** If you want to use the latest release with your WordPress site, <a href="https://wordpress.org/plugins/gutenberg/">download the latest release from the WordPress.org plugins repository</a>.
-- **Discuss:** Conversations and discussions take place in <a href="https://wordpress.slack.com/messages/C02QB2JS7">`#core-editor` channel on the Making WordPress Slack</a>.
-- **Contribute:** Development of Gutenberg happens in this GitHub repo. Get started by <a href="https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md">reading the contributing guidelines</a>.
-- **Learn:** <a href="https://wordpress.org/gutenberg/">Discover more about the project on WordPress.org</a>.
-
-**Gutenberg is more than an editor.** While the project is currently focused on building the new editor for WordPress, it doesn't end there. This lays the groundwork for a new model for WordPress Core that will ultimately impact the entire publishing experience of the platform.
-
-## Editing focus
-
-> *The editor will create a new page- and post-building experience that makes writing rich posts effortless, and has “blocks” to make it easy what today might take shortcodes, custom HTML, or “mystery meat” embed discovery.*
->
-> — Matt Mullenweg
+Gutenberg is more than an editor. It is the foundation that will revolutionize customization, site building and the entire publishing experience of the platform. The four phases of the project are Editing, Customization, Collaboration, and Multilingual.
 
-One thing that sets WordPress apart is that it allows you to create a post layout that's as rich as you can imagine—but only if you can build your own custom theme with HTML and CSS. By thinking of the editor as a tool that allows you to write rich posts **and** create beautiful layouts, we can transform WordPress into something users _love_, as opposed to something they choose because it happens to be what everyone else uses.
+Gutenberg development precedes what is shipped in a WordPress releases, usually by numerous weeks. The latest development version of Gutenberg is available as a plugin for testing and trying out the latest features.
```

Comment:
This phrase has a couple of ingredients that could be used in the first paragraph, as I just suggested. But I would still use the term "Gutenberg" less often, if at all possible, even if this is the gutenberg repo.